test/ct.yaml

```diff
@@ -5,4 +5,5 @@ chart-dirs:
   - proposed
 excluded-charts:
 chart-repos:
+  - stable=https://charts.helm.sh/stable
```

Comment:
https://github.com/helm/chart-testing/issues/316#issuecomment-762538679

Reply:
Will address in a separate PR to speed things up since it could potentially fail the chart because the image is super old (I've seen it happen before). I've linted the charts before with the latest version locally and they are good.